core/snippets/snippets/import.snippet

```diff
@@ -0,0 +1,29 @@
+name: importStatement
+phrase: import
+
+---
+
+language: javascript | typescript | javascriptreact | typescriptreact
+-
+import $0 from "$0";
+---
+
+language: python
+-
+import $0
+---
+
```

Comment:
I'm not sure why, but this is failing to parse with the addition of the python snippet

```
2025-03-07 22:46:14.509    IO ERROR | import.snippet:15 | Missing body in snippet document '
name: importStarStatement
phrase: import star

'
```

Reply:
You can only have one header per file.

Reply:
Ah, so we need to split the import star into its own file.

Reply:
That or don't use a header for the file at all. It's perfectly valid to have multiple different snippets in one file, but then I wouldn't use a header because they don't have enough in common. In this particular case I would recommend a separate file.